server/src/models/courseTask.ts

```diff
@@ -17,6 +17,7 @@ import { Course } from './course';
 import { TaskSolution } from './taskSolution';
 
 export enum Checker {
+  AutoTest = 'auto-test',
```

Comment:
5 cents: it would be nice to unify codestyle here, we have camel case and snake case in the same enum

Reply:
agree but it would require data migrations and i don't want to deal with it now